dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java

```diff
@@ -13,19 +13,41 @@
     private Date dbUpdateDate;
     private long id;
     private String entryType;
+    private String versionName;
 
     public CollectionEntry(long id, Date dbUpdateDate, String entryTypeString, SourceControl sourceControl, String organization, String repository, String entryName)  {
         setEntryType(entryTypeString);
         setDbUpdateDate(dbUpdateDate);
         setId(id);
         setEntryPath(sourceControl.toString(), organization, repository, entryName);
+        setVersionName(null);
     }
 
+    @SuppressWarnings("checkstyle:ParameterNumber")
```

Comment:
Are these different constructors with different parameters null? 
Might be able to re-use, for example 
`CollectionEntry(long id, Date dbUpdateDate, String entryTypeString, SourceControl sourceControl, String organization, String repository, String entryName)`
can call 
`this(id, dbUpdateDate, entryTypeString, sourceControl, organization, repository, entryName, null))`

(Typos above probably)
Other example https://stackoverflow.com/questions/1182153/constructor-overloading-in-java-best-practice